README.md

```diff
@@ -1,9 +1,26 @@
-# This is a template useful to create guides
+# User guides for the OMERO R API
+
+This guide demonstrates how to use the OMERO R API.
+
+To run the notebooks, you can either [run on mybinder.org](https://mybinder.org/v2/gh/ome/omero-guide-r/master?filepath=notebooks) or build locally with [repo2docker](https://repo2docker.readthedocs.io/).
+
+
+To build locally:
+
+ * Install [Docker][https://www.docker.com/] if required
+ * Create a virtual environment and install repo2docker from PyPI.
```

Comment:
@manics is recommending to install from master instead of pip install repo2docker - should we recommend it here ?

Reply:
i am a bit reluctant to install from master in our doc
I have now used the installation from PyPI several times and I did not have any problems
What is the reason for following master?